File: Go-Standalone/test_ws.go
package main

import (
	"encoding/json"
	"fmt"
	"github.com/gorilla/websocket"
)

func main() {
	uri := "wss://gateway.discord.gg/?v=10&encoding=json"
	ws, _, err := websocket.DefaultDialer.Dial(uri, nil)
	if err != nil {
		fmt.Println("Dial error:", err)
		return
	}
	defer ws.Close()

	_, msg, _ := ws.ReadMessage()
	fmt.Println("Hello:", string(msg))

	payload := map[string]interface{}{
		"op": 2,
		"d": map[string]interface{}{
			"token":   "DISCORD_TOKEN_HERE",
			"intents": 3276799,
			"properties": map[string]string{
				"os":      "linux",
				"browser": "ezbot",
				"device":  "ezbot",
			},
		},
	}
	data, _ := json.Marshal(payload)
	ws.WriteMessage(websocket.TextMessage, data)

	_, resp, err := ws.ReadMessage()
	fmt.Println("Identify response:", string(resp), err)
}

File: Go-Standalone/test_ws.py
import json
import asyncio
import websockets

async def test():
    token = "DISCORD_TOKEN_HERE"
    uri = "wss://gateway.discord.gg/?v=10&encoding=json"
    async with websockets.connect(uri) as ws:
        hello = await ws.recv()
        print("Hello:", hello)
        
        payload = {
            "op": 2,
            "d": {
                "token": token,
                "intents": 33281, # Guilds | GuildMessages | MessageContent
                "properties": {
                    "os": "linux",
                    "browser": "ezbot",
                    "device": "ezbot"
                }
            }
        }
        await ws.send(json.dumps(payload))
        
        while True:
            try:
                res = await ws.recv()
                print("Recv:", res)
            except Exception as e:
                print("Error:", e)
                break

asyncio.run(test())
